latest-thread.html

```diff
@@ -0,0 +1,13 @@
+---
+layout: page
+title: Latest LOOT thread
+permalink: /latest-thread/
+redirect_to:
+  - http://forums.bethsoft.com/topic/1588424-rel-loot-thread-22/
+---
+
+<p>The latest thread about LOOT on the Bethesda forums can be found at:
```

Comment:
The redirect is done using a `<meta http-equiv="refresh" content="…">` tag (https://en.wikipedia.org/wiki/Meta_refresh). These can be blocked by some browser extensions or otherwise disabled for/by users (I think it may, in some browsers at least, also not redirect if they stop loading the page before it's been fully loaded), so it's a good idea to actually provide some content so people ending up on the page and not getting redirected do actually still have the ability to continue to the thread instead of ending up on a blank page.